UtopiaCity/Services/Airport/FlightRouteApiService.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+using UtopiaCity.Models.Airport;
+
+namespace UtopiaCity.Services.Airport
+{
+    public class FlightRouteApiService : IRouteApi
+    {
+        private static readonly HttpClient client;
```

Comment:
I am not sure it should be static. Is there any reason?

Reply:
it won't work in constructor if i remove the static , should i implement this through DI pattern ?

Reply:
mm understand , will remove the modificator , i did the constructor static because of some advises from Microsoft.Docs , but i realised that they not really in my scope